src/OneDriveSdk/Requests/Extensions/OneDriveClientExtensions.cs

```diff
@@ -268,11 +366,12 @@ public partial class OneDriveClient : IDisposable
         /// <param name="returnUrl">The application return URL for Microsoft account authentication.</param>
         /// <param name="scopes">The requested scopes for Microsoft account authentication.</param>
         /// <param name="clientSecret">The client secret for Microsoft account authentication.</param>
+        /// <param name="refreshToken">The refresh token for silent authentication.</param>
         /// <param name="serviceInfoProvider">The <see cref="IServiceInfoProvider"/> for initializing the <see cref="IServiceInfo"/> for the session.</param>
         /// <param name="credentialCache">The cache instance for storing user credentials.</param>
         /// <param name="httpProvider">The <see cref="IHttpProvider"/> for sending HTTP requests.</param>
         /// <returns>The <see cref="IOneDriveClient"/> for the session.</returns>
-        public static async Task<IOneDriveClient> GetSlientlyAuthenticatedMicrosoftAccountClient(
+        public static async Task<IOneDriveClient> GetSilentlyAuthenticatedMicrosoftAccountClient(
```

Comment:
I found a tool to spell check whole words but not method or class names. Including those fixes, though.